# Informatievisualisatie: Data story project

Universiteit van Amsterdam - Juni 2023 

Jarno Balk: 14382385 

Hamza Nur: 13978071

Luca van Kessel: 14413744

Iyoas Ghebreab: 

Onderwerp: Oorzaken criminaliteit


# _Oorzaken criminaliteit_

# 1. Introductie
In de afgelopen 20 jaar is er volgens verschillende rapporten van de Verenigde Naties een wereldwijde toename van criminaliteit.  De toename van criminaliteit vindt plaats op alle niveaus, van straatcriminaliteit tot georganiseerde misdaad (“World crime trends and emerging issues and responses in the field of crime prevention and criminal justice”, 2015). Criminaliteit heeft enorme impact op de individuele veiligheid van burgers in een land maar kan ook een heel land in de greep houden. In het jaar 2020 vond er in de criminaliteit een wereldwijde toename plaats van 0,75 procent. Toch is er een substantieel verschil in de soorten delicten die er gepleegd worden per werelddeel. Zuid-Amerika heeft veel te kampen met drugscriminaliteit terwijl Noord-Amerika veel te kampen heeft met wapengeweld in grote steden. Ieder jaar wordt er een crime index opgesteld hierin wordt is het aantal criminele activiteiten in een land berekend ten opzichte van het aantal inwoners. Figuur 1 bevat 10 landen wereldwijd en hierin is de crime index van het jaar 2020 berekend. Deze 10 landen zijn geselecteerd op inwonersaantal en op continent. Op de x-as bevinden zich de landen en op de y-as de waarden van de crime index.

In [28]:
import pandas as pd
import plotly.graph_objects as go

# Read the CSV file
data = pd.read_csv('crime_index_by_country (Introductie).csv')

# Get the column names containing the data for each chart
chart_columns = data.columns[1:]

# Generate bar charts for each column
for column in chart_columns:
    # Extract country names from the 'Country' column
    countries = data['Country'].tolist()

    # Get the values for the current column
    values = data[column].tolist()

    # Create a bar chart using Plotly
    fig = go.Figure(data=[go.Bar(x=countries, y=values)])
    fig.update_layout(xaxis_tickangle=-45, xaxis_title='Country', yaxis_title='Crime Index', title=f'Figuur 1: {column}')
    fig.show()


Er zijn verschillende oorzaken die invloed kunnen hebben op de criminaliteit zoals maatschappelijke en economische redenen. Ongelijkheid, armoede, werkloosheid, gebrek aan onderwijskansen, sociale ongelijkheid en gebrekkige toegang tot hulp kunnen allemaal bijdragen aan de toename van criminaliteit. De Human Development Index (Hierna: HDI) berekent de mate waarin een land op verschillende factoren is ontwikkeld zoals onderwijs en kansengelijkheid. De HDI bevat een getal tussen de 0 en 1000 waarbij 0 'zwak ontwikkeld' is en 1000 'zeer hoog ontwikkeld' is. Figuur 2 bevat 10 landen wereldwijd hierin is de HDI van het jaar 2020 berekend. Deze 10 landen zijn geselecteerd op inwonersaantal en op continent. Op de x-as bevinden zich de landen en op de y-as de waarden van de HDI.

In [29]:
import pandas as pd
import plotly.express as px

# Read the CSV file
data = pd.read_csv('HDI(introduction).csv')

# Get the column names containing the data for each chart
chart_columns = data.columns[1:]

# Generate bar charts for each column
for column in chart_columns:
    # Extract country names from the 'Country' column
    countries = data['Country'].tolist()

    # Get the values for the current column
    values = data[column].tolist()

    # Create a bar chart using Plotly
    fig = px.bar(x=countries, y=values, labels={'x': 'Country', 'y': 'Human Development Index'}, title=f'Figuur 2: {column}')
    fig.update_layout(xaxis_tickangle=-45)
    fig.show()


In dit artikel zullen twee perspectieven aan bod komen. Ten eerste een sociaal-maatschappelijk perspectief dat stelt dat criminaliteit wordt veroorzaakt door verschillen in onderwijs en levensverwachting. Ten tweede een economisch perspectief dat stelt dat de welvaart van een land en haar inwoners de hoofdoorzaak is. Beide perspectieven zullen in dit artikel aan de hand van de HDI en de crime index worden gevisualiseerd.  

# 2. Datasets en voorbereiding

Om de verschillende verbanden te analyseren, is er gebruik gemaakt van twee datasets namelijk de 'Crime Index' en de 'HDI'. Beide datasets zijn opgeslagen in verschillende CSV-bestanden.

De 'Crime Index' dataset bevat informatie over criminaliteit en veiligheid in een land. De kolom 'Country' vermeldt de namen van verschillende landen. De variabele 'Crime Index' kwantificeert het niveau van criminaliteit in elk land, waarbij hogere waarden duiden op hogere criminaliteitscijfers. De 'Safety Index' vertegenwoordigt het veiligheidsniveau in elk land, waarbij hogere waarden duiden op veiligere omgevingen. Na het inlezen van het CSV-bestand met behulp van de pandas library, zijn de relevante variabelen uit de dataset geselecteerd. 
 
Om het verband tussen bijvoorbeeld levensverwachting en criminaliteit te analyseren, is er gebruik gemaakt van twee datasets. De 'Crime Index' dataset bevat informatie over de criminaliteit in verschillende landen. Deze dataset bevat de variabel 'Country', die de naam van elk land vertegenwoordigt, en 'Crime Index', die het niveau van criminaliteit in elk land kwantificeert.

Hieronder bevindt zich de 'dataframe head' van de crime dataset. Dit is een greep uit de dataset en geeft een beeld van hoe de dataset is opgebouwd.

In [30]:
import pandas as pd

# Read the CSV file
filename = 'crime_index_by_country.csv'  # Replace with the actual file name or file path
data = pd.read_csv(filename)

# Display the dataframe's head
print(data.head())


            Country  Crime Index  Safety Index
0         Venezuela        84.36         15.64
1  Papua New Guinea        80.04         19.96
2      South Africa        77.29         22.71
3       Afghanistan        76.97         23.03
4          Honduras        76.65         23.35


De tweede dataset is de 'Human Development Index', de HDI bevat veel gegevens over de menselijke ontwikkeling in een land, waaronder levensverwachting en onderwijsjaren. De HDI bevat een getal tussen 0 en 1000. Hoe verder een land is ontwikkeld, hoe dichter het cijfer bij 1000 komt. 

Om het verband tussen levensverwachting en criminaliteit te analyseren, zijn de relevante gegevens uit beide datasets geselecteerd. De variabelen 'Country' en 'Crime Index' zijn uit de 'Crime Index' dataset geselecteerd, evenals 'Country' en 'Life Expectancy at Birth (2020)' uit de HDI dataset. Vervolgens zijn de datasets samengevoegd op basis van de overeenkomstige landen. Voor het verband tussen criminaliteit en onderwijsjaren is er gebruik gemaakt van de variabelen 'Mean Years of Schooling (2020)' en 'Crime Index'. Door middel van de 'Crime Index' dataset' en 'HDI' dataset wordt het verbanden tussen criminaliteit, bnp per hoofd en inkomensongelijkheid aangetoond. Deze variabelen zijn terug te vinden in de HDI dataset. Om het verband tussen de verschillende variabelen te bepalen is ook de correlatiecoëfficiënt berekend. 
 
Hieronder bevindt zich de 'dataframe head' van de HDI dataset. Dit is een greep uit de dataset en geeft een beeld van hoe de dataset is opgebouwd. De HDI dataset is een grote dataset en daarom is er hieronder een weergave van 10 kolommen.

In [31]:
import pandas as pd

# Read the CSV file
data = pd.read_csv('Human Development Index - Full.csv')

# Display the dataframe head for the first 10 columns
columns_to_display = data.columns[:10]
head_data = data[columns_to_display].head()
print(head_data)



  ISO3               Country Human Development Groups UNDP Developing Regions  \
0  AFG           Afghanistan                      Low                      SA   
1  AGO                Angola                   Medium                     SSA   
2  ALB               Albania                     High                     ECA   
3  AND               Andorra                Very High                     NaN   
4  ARE  United Arab Emirates                Very High                      AS   

   HDI Rank (2021)  Human Development Index (1990)  \
0            180.0                           0.273   
1            148.0                             NaN   
2             67.0                           0.647   
3             40.0                             NaN   
4             26.0                           0.728   

   Human Development Index (1991)  Human Development Index (1992)  \
0                           0.279                           0.287   
1                             NaN                 

# 3. Perspectieven
# 3.1 Sociaal-maatschappelijk


**3.1.1 De rol van onderwijs in de criminaliteit**

Uit vele onderzoeken is gebleken dat onderwijs een belangrijke rol speelt bij het bestrijden van criminaliteit.  Zo ook uit een onderzoek verricht door de Universiteit Maastricht in samenwerking met de Universiteit van Amsterdam (Groot & Van Den Brink, 2010). Onderwijs heeft een belangrijke invloed op het levenspatroon van (jonge) mensen. Uit het onderzoek is gebleken dat de kans op criminaliteit lager is bij landen die goed investeren in onderwijs. Ook is gebleken dat goed onderwijs ervoor zorgt dat naar mate het aantal jaar dat (jonge) mensen naar school gaan toeneemt, het aantal criminele handelingen zoals fraude, overvallen, vandalisme en bedreigingen afneemt. Daarnaast blijkt hoe hoger mensen opgeleid zijn hoe meer bewust ze zijn van effecten van criminaliteit. Figuur 3 bevat twee variabelen namelijk het gemiddeld aantal onderwijsjaren in een land en de crime index van het land. 

In [32]:
import pandas as pd
import plotly.express as px

# Read the CSV file
data = pd.read_csv('crime_index_by_country_HDI(Education).csv')

# Create bar plots using Plotly Express
fig = px.bar(data, x='Country', y=['Crime Index', 'Mean Years of Schooling (2020)'], barmode='group')

# Customize the plot
fig.update_layout(
    xaxis_title='Country',
    yaxis_title='Value',
    title='Figuur 3: Crime Index and Mean Years of Schooling',
)

# Show the plot
fig.show()

Uit de bovenstaande staafdiagrammen blijkt dat hoe lager het gemiddeld aantal jaren dat mensen naar school gaan in een land, hoe hoger de criminaliteitsgraad. Op de x-as vindt zich de namen van de landen. Op de Y-as de waarden. Een voorbeeld uit het diagram waarbij er een hoog aantal onderwijsjaren zijn is China, in China gaan mensen gemiddeld 7.6 jaar naar school. In China ligt de crime rate op 31.18. Een voorbeeld uit de grafiek waarbij er een laag aantal onderwijsjaren zijn is Papua New Guinea, hier gaan mensen 4.74 jaar naar school. De crime index ligt hier op 80.04. In figuur 3 zijn er 2 bewegingen te zien hoe lager de criminaliteitsgraad hoe hoger het gemiddeld aantal onderwijsjaren. Deze beweging is ook andersom waarneembaar, hoe hoger de criminaliteitsgraad hoe lager het aantal onderwijsjaren. Er kan dus met succes worden geconcludeerd dat er een verband is tussen het aantal onderwijsjaren en de criminaliteitsgraad in een land. 

**3.1.2 De rol van de levensverwachting van mensen in de criminaliteit**

In de context van het onderzoek naar criminaliteit en sociale indicatoren is het begrijpen van de rol van levensverwachting van cruciaal belang. Deze statistische maatstaf is een belangrijke indicator van de algemene gezondheid en welzijn van een populatie. In dit onderdeel wordt de correlatie tussen de Crime Index en de Levensverwachting bij Geboorte onderzocht. De Crime Index geeft inzicht in het niveau van criminaliteit in verschillende landen, terwijl de Levensverwachting bij Geboorte een indicator is van de gemiddelde levensduur van mensen in die landen. De correlatie tussen deze twee variabelen kan waardevolle inzichten bieden in de mogelijke relatie tussen criminaliteit en gezondheid. Dit onderzoek is gebaseerd op gegevens die beschikbaar zijn voor een specifiek jaar, namelijk het jaar waarin de Levensverwachting bij Geboorte is gemeten.

In [33]:
import pandas as pd
import plotly.express as px
import numpy as np

# Laad de gegevens
file_path = 'crime_index_by_country.csv'
file_path2 = 'Human Development Index - Full.csv'
crime_index = pd.read_csv(file_path)
hdi_index = pd.read_csv(file_path2)

# Selecteer relevante kolommen
crime_data = crime_index[['Country', 'Crime Index']]
hdi_data = hdi_index[['Country', 'Life Expectancy at Birth (2020)']]

# Combineer datasets op basis van landenamen
merged_data = pd.merge(crime_data, hdi_data, on='Country', how='inner')

# Bereken de correlatie
correlation = merged_data['Crime Index'].corr(merged_data['Life Expectancy at Birth (2020)'])

# Print de correlatiecoëfficiënt
print("Correlatie tussen Crime Index en Levensverwachting bij Geboorte:", correlation)

# Maak een spreidingsdiagram
fig = px.scatter(merged_data, x='Life Expectancy at Birth (2020)', y='Crime Index',
                 labels={'Life Expectancy at Birth (2020)': 'Levensverwachting bij Geboorte',
                         'Crime Index': 'Crime Index'},
                 title='Spreidingsdiagram Crime Index vs. Levensverwachting bij Geboorte (2020)')

# Voeg trendlijn toe
x = merged_data['Life Expectancy at Birth (2020)']
y = merged_data['Crime Index']
coefficients = np.polyfit(x, y, 1)
trendline_x = np.array([min(x), max(x)])
trendline_y = np.polyval(coefficients, trendline_x)
fig.add_trace(px.line(x=trendline_x, y=trendline_y).data[0])

# Toon de grafiek
fig.show()


Correlatie tussen Crime Index en Levensverwachting bij Geboorte: -0.5547227165122521


De correlatiecoëfficiënt van -0.5547227165122521 tussen de Crime Index en de Levensverwachting bij Geboorte geeft aan dat er een matige negatieve correlatie bestaat tussen deze twee variabelen. Deze observatie wordt ondersteund door de spreidingsdiagram met de trendlijn die weergegeven wordt. Op de plot is te zien dat naarmate de levensverwachting bij geboorte toeneemt, de crime index over het algemeen afneemt, en vice versa.

De trendlijn op de plot illustreert de algemene trend tussen de Crime Index en de Levensverwachting bij Geboorte. De neerwaartse richting van de trendlijn bevestigt de negatieve correlatie. Het suggereert dat landen met hogere criminaliteitscijfers doorgaans lagere levensverwachtingen bij geboorte hebben, terwijl landen met lagere criminaliteitscijfers een hogere levensverwachting bij geboorte laten zien.

Het is echter belangrijk om op te merken dat correlatie geen causaal verband impliceert. Andere factoren kunnen van invloed zijn op de relatie tussen criminaliteit en levensverwachting bij geboorte. Het vereist verder onderzoek en analyse om de specifieke mechanismen en oorzaken achter deze waarnemingen te identificeren.

Niettemin biedt deze correlatie ons waardevol inzicht in de mogelijke interactie tussen criminaliteit en de gezondheid en welzijn van een samenleving. Het onderstreept het belang van het bevorderen van veiligheid en het aanpakken van criminaliteit als een middel om de algemene gezondheid en het welzijn van de bevolking te verbeteren.

# 3.2 Economisch

**3.2.1 De rol van de economie in de criminaliteit**

In recente jaren zijn we als wereldeconomie een zekere recessie aan het betreden. De prijzen van alles gaat omhoog. Eten, wonen, brandstof. Alles wordt duurder. Dit heeft een grote impact op iedereen. Al het geld wordt minder waard, en dit heeft een psychologische impact op ons allen. De onrust die de recessie met zich meebrengt maakt mensen onvoorspelbaar en prikkelbaar. Mede door de coronapandemie is de kloof tussen rijk en arm in vrijwel elk land gegroeid. Er is een algemene consensus bereikt dat criminaliteit onder andere door financiële moeilijkheden wordt veroorzaakt. Als een persoon niet rond kan komen op de legale manier, zal hij zich niet laten stoppen door de regels. Een persoon moet immers kunnen eten en zorgen voor zijn naasten. Daarnaast is een motivator voor criminaliteit, voornamelijk diefstal, het gevoel dat iets ontbreekt wat iemand anders wel heeft. In landen waarbij de inkomensverschillen groot zijn, zal er ook vaker meer criminaliteit zijn.

In [34]:
import pandas as pd
# Load dataframes
df_crime= pd.read_csv("crime_index_by_country.csv")
df_hdi= pd.read_csv("Human Development Index - Full.csv")

# Aparte dataframe voor de gebruikte variabelen gesorteerd op crime index
df_plot = pd.DataFrame()
df_plot['Income Inequality'] = df_hdi['Inequality in income (2020)'].dropna()
df_plot['BNP/hoofd'] = df_hdi['Gross National Income Per Capita (2020)'].dropna()
df_plot['HDI'] = df_hdi['Human Development Index (2020)'].dropna()
df_plot['Country'] = df_hdi['Country']
df_plot_final = df_plot.merge(df_crime, on='Country', how='inner').sort_values(by='Crime Index', ascending=False).reset_index().drop(columns=['index', 'Safety Index'])


In [35]:
print(df_plot_final['BNP/hoofd'].corr(df_plot_final['Crime Index']))
print(df_plot_final['Income Inequality'].corr(df_plot_final['Crime Index']))
import plotly.graph_objects as go


trace = go.Scatter3d(
    x=df_plot_final['BNP/hoofd'],
    y=df_plot_final['Income Inequality'],
    z=df_plot_final['Crime Index'],
    mode='markers',
    marker=dict(
        size=6,  
        color='red',
        symbol='circle',
        line=dict(color='black', width=0.5)
    ),
    text=df_plot_final['Country']
)


layout = go.Layout(
    title='Landen en hun Crime Index, inkomensongelijkheid en BNP per hoofd (2020)',
    scene=dict(
        xaxis=dict(title='BNP/hoofd'),
        yaxis=dict(title='Income Inequality'),
        zaxis=dict(title='Crime Index'),
    ),
    margin=dict(l=0, r=0, b=0, t=40),
    height=700,  
    width=900,  
)


fig = go.Figure(data=[trace], layout=layout)


fig.show()

-0.5390810718243931
0.5395336186949828


De bovenstaande visualisatie geeft een intrigerend beeld van de relatie tussen de variabelen crime index, BNP per hoofd en inkomensongelijkheid in verschillende landen. De y-as toont de Crime Index, de x-as toont de inkomensongelijkheid en de z-as toont het BNP per hoofd.

Een gemiddelde tot sterke inverse relatie tussen het BNP per hoofd en de Crime Index wordt aangetoond door de negatieve correlatiecoëfficiënt van -0.5390810718243931. Dit betekent dat de Crime Index meestal afneemt naarmate het BNP per persoon toeneemt. Het suggereert dat hogere BNP per hoofd over het algemeen lagere criminaliteitscijfers hebben. Dit resultaat kan de hypothese ondersteunen dat economische welvaart en ontwikkeling verband houden met een afname van criminaliteit.

De positieve correlatiecoëfficiënt van 0.5395336186949828 tussen de Crime Index en inkomensongelijkheid laat echter zien dat er een relatie is die gemiddeld tot sterk positief is. Dit betekent dat de Crime Index stijgt naarmate de inkomensongelijkheid toeneemt. Het wijst er op dat criminaliteit hoger is in landen met grotere inkomensongelijkheid. 

Bij het bekijken van de plot valt op dat de Crime Index stijgt vanaf een inkomensongelijkheid van ongeveer dertig. De Crime Index komt dan boven de 40 uit, wat wordt geclassificeerd als een hoog crimineel niveau. Dit geeft aan dat er een verband bestaat tussen criminaliteit en inkomensongelijkheid; hogere inkomensongelijkheid wordt geassocieerd met een grotere kans op criminaliteit.

Bovendien kunnen we uit de plot afleiden dat alle landen met een Crime Index van hoger dan 48 voorkomen bij een BNP per hoofd van onder de $23.000. Dit suggereert dat landen met een lager BNP per hoofd en een hogere Crime Index mogelijk een groter risico hebben op criminaliteit.

Kortom, de gegevens laten zien dat landen met hogere inkomensongelijkheid en lagere BNP per hoofd waarschijnlijk hogere criminaliteitsniveaus hebben. Bovendien kunnen we uit de gegevens afleiden dat hogere inkomensongelijkheid gepaard gaat met hogere criminaliteit in een land. 

**3.2.2 De rol van de Index van de Menselijke Ontwikkeling in de criminaliteit**

In [36]:
import pandas as pd
import plotly.express as px

crime_index_file = 'crime_index_by_country.csv'
HDI_File = 'Human Development Index - Full.csv'
crime_index = pd.read_csv(crime_index_file)
hdi_index = pd.read_csv(HDI_File)

# data selecteren
crime_data = crime_index[['Country', 'Crime Index']]
hdi_data = hdi_index[['Country', 'Human Development Index (2020)']]

# Datasets combineren
merged_data = pd.merge(crime_data, hdi_data, on='Country', how='inner')

# De heatmap
fig = px.density_heatmap(merged_data, x='Human Development Index (2020)', y='Crime Index',
                         labels={'Human Development Index (2020)': 'Human Development Index',
                                 'Crime Index': 'Crime Index'},
                         title='Crime Index vs. Human Development Index Heatmap (2020)')

# Bereken de correlatie
correlation = merged_data['Crime Index'].corr(merged_data['Human Development Index (2020)'])

# Correlatie laten zien in de grafiek
description = "correlatiecoëfficiënt: {}".format(correlation)
fig.add_annotation(text=description, xref="paper", yref="paper", x=1.055, y=-0.25, showarrow=False, font=dict(size=13))

fig.show()

Bovenstaande heatmap toont een duidelijk verband tussen de Human Development Index (HDI) en de criminaliteitsindex. De negatieve correlatie 
-0.5738010593076646 laat duidelijk zien dat er een negatief verband bestaat. Hogere HDI-waarden worden geassocieerd met lagere criminaliteitsniveaus, terwijl lagere HDI-waarden gepaard gaan met hogere criminaliteitsniveaus. Dit onderstreept het belang van een hogere HDI bij het verminderen van criminaliteit en vice versa.

Mogelijk komt dit doordat mensen in landen met een lage menselijke ontwikkeling een slechter toekomstbeeld hebben en sneller de criminele weg inslaan in de hoop op verbetering. Dit benadrukt het belang van investeren in gezondheid, onderwijs en levensstandaard als strategie om criminaliteit te verminderen.

Een hogere HDI zorgt voor een lagere crime index, mede omdat landen met een betere menselijke ontwikkeling over het algemeen betere sociale en economische kansen bieden aan hun mede inwoners. Dit vermindert de motivatie en noodzaak voor mensen om zich bezig te houden met criminele activiteiten.

Kortom, het is duidelijk volgens de heatmap en de correlatie coëfficient dat er een negatief verband zit tussen de menselijke ontwikkeling en de criminaliteitsindex. Ook is het volgens de argumentatie een logische conclusie om te trekken. Dit laat zien dat zowel de economische aspecten, de levensverwachting en het onderwijs allemaal een rol spelen in de criminaliteitscijfers van een land.

# 4. Reflectie

Naar aanleiding van de feedback presentatie van 27 juni zijn er een aantal aanpassingen gedaan aan dit artikel. In eerste instantie waren alle plots gemaakt in matplotlib. Op aanraden van medestudenten en de TA is dit aangepast naar plotly. Dit geeft meteen een betere en heldere weergave van de analyses. Daarnaast is er bij 'datasets en voorbereiding' een datahead frame toegevoegd. Hierdoor krijgt de lezer te zien hoe de datasets zijn opgebouwd.

Het eerste perspectief werd aanvankelijk weergeven met een lijn grafiek. Dit is vervangen door een staafdiagram omdat het hier gaat om telkens 2 categorieën namelijk het aantal jaren onderwijs en de criminaliteitsgraad. De staafdiagram weergeeft de hoogte van de variabelen op overzichtelijke wijze. 

Bij de spreidingsdiagram over levensverwachting en criminaliteit is een
trendlijn toegevoegd om de relatie tussen de variabelen duidelijker weer te geven. Ook is er
meer tekst toegevoegd die wordt ondersteund door het plaatje, waardoor de bevindingen beter worden toegelicht.
De toevoeging van de trendlijn helpt bij het visualiseren van de algemene trend tussen de Crime Index en de Levensverwachting bij Geboorte. Dit biedt een visuele ondersteuning voor de matige negatieve correlatie die wordt beschreven. Bovendien is er meer tekst toegevoegd die specifiek verwijst naar het plaatje en de observaties ervan, waardoor de bevindingen beter worden onderbouwd.

Daarnaast is de diagram met de variabelen BNP/hoofd, income inequality en crime index aangepast. Deze diagram was aan het begin wat onoverzichtelijk door de slechte representaties van de 3 variabelen in 1 diagram. Hiervoor is een 3D-scatterplot in de plaats gekomen. Dit geeft een beter inzicht in de variabelen en heeft een interactiemogelijkheid voor de lezer. Het overzicht is nu duidelijker te zien, waardoor het makkelijker werd om een argumentatie erover te geven.

Als laatste is de scatterplot van de HDI en de crime index veranderd. Deze was echter te simpel en toonde de correlatie te matig, waardoor het te onduidelijk was. Hiervoor is nu een heatmap in de plaats gekomen, die de correlatie beter aangeeft en daardoor de argumentatie versterkt. Ook zijn de bevindingen over de heatmap breder uitgelegd, zodat het verhaal duidelijker wordt.

# 5. Taakverdeling

De taken voor dit project zijn gelijkmatig verdeeld. Alle teamleden hebben gewerkt met beide datasets. De perspectieven zijn opgedeeld in meerdere argumenten per perspectief. Hamsa en Iyoas zijn voornamelijk bezig geweest met het eerste perspectief namelijk het sociaal-maatschappelijke perspectief. De inleiding en bijbehorende figuren is geschreven door Hamsa en de 'Datasets en voorbereiding' is geschreven door Iyoas. Het eerste perspectief kent 2 argumenten, het eerste argument (3.1.1) is geschreven door Hamsa en het tweede argument (3.1.2) is geschreven door Iyoas. 

Jarno en Luca hebben gewerkt aan het tweede argument met betrekking tot het economische perspectief. Luca heeft het eerste argument van het tweede perspectief geschreven (3.1.3) en Jarno het tweede argument van het tweede perspectief (3.1.4). Tot slot is de reflectie en bronnenlijst het werk van alle teamleden.

# 6. Bronnen

**Introductie**

World crime trends and emerging issues and responses in the field of crime prevention and criminal justice. (2015). _In Official records (pp. 92–95). United Nations._ https://doi.org/10.18356/76a7e412-en

**Perspectief 1 (argument 1)**

Groot, W., & Van Den Brink, H. M. (2010). _The effects of education on crime. Applied Economics, 42(3), 279–289._ https://doi.org/10.1080/00036840701604412

**Gehele opdracht**

_Chatgpt_. (2023). https://chat.openai.com